.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 defaults:
-  default-xcode-version: &default-xcode-version "12.2.0"
+  default-xcode-version: &default-xcode-version "13.3.1"
```

Comment:
Is this change still necc. ? If not lets revert this to 12 and make a future work ticket to **add** xcode 13 support for CICD. which means running some things twice once for each version.